Gruntfile.js

```diff
@@ -383,6 +383,23 @@ module.exports = function (grunt) {
             src: ['**', '!hmda-pilot.zip']
           }
         ]
+      },
+      'codedeploy': {
```

Comment:
If I'm following it right, looks like the `compress:hmda-pilot` zips up the actual deployable source code, and `compress:codedeploy` zips the created `hmda-pilot.zip` from the previous along with support files that codedeploy needs to instruct it how to actually deploy the `hmda-pilot.zip`